src/VisualStudio/Core/SolutionExplorerShim/AnalyzerReferenceManager.cs

```diff
@@ -89,9 +92,19 @@ public void ChangeReferences(uint operation, IVsReferenceProviderContext changed
         public Array GetProviderContexts()
         {
             // Return just the File provider context so that just the browse tab shows up.
-            var context = _referenceManager.CreateProviderContext(VSConstants.FileReferenceProvider_Guid) as IVsFileReferenceProviderContext;
+            var context = GetReferenceManager().CreateProviderContext(VSConstants.FileReferenceProvider_Guid) as IVsFileReferenceProviderContext;
```

Comment:
It shouldn't be necessary. This method is called by the reference manager after we tell it to show the dialog; in that case, the reference manager shouldn't be null.